graphs/0035_GraphNode_Basic.json

```diff
@@ -30,7 +30,7 @@
       "component": "rdf-components/RequestTemplate",
       "metadata": {
         "label": "LOAD INTO GRAPH",
-        "x": 324,
+        "x": 288,
```

Comment:
Would like to see us move this graph 35 json file over to the noflo-rdf-pipeline/tests since it really should belong with David's legacy tests there.